pages/myProfile/userCounter/userCounter.tsx

```diff
@@ -0,0 +1,15 @@
+import s from './userCounter.module.scss';
+
+type UserCounterType = {
```

Comment:
The `Type` part of `UserCounterType` is undesirable because it is already __type__. You can write simply `UserCounter`. So avoid duplicating the word 'type' here and in the rest of the code.